.github/workflows/ci.yml

```diff
@@ -1,5 +1,11 @@
 name: CI
-on: push
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Why limit this?

Reply:
No particular reason. We can relax this when necessary. For now it should cover the current flow.